flutter/news/test/features/frontpage/presentation/bloc/articles_cubit_test.dart

```diff
@@ -0,0 +1,64 @@
+import 'package:bloc_test/bloc_test.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:mockito/annotations.dart';
+import 'package:mockito/mockito.dart';
+import 'package:news/core/error/failures.dart';
+import 'package:news/core/result.dart';
+import 'package:news/features/frontpage/data/repositories/articles_repository.dart';
+import 'package:news/features/frontpage/domain/entities/article.dart';
+import 'package:news/features/frontpage/domain/entities/source.dart';
+import 'package:news/features/frontpage/presentation/bloc/articles_cubit.dart';
+
+import '../../domain/usecases/get_everything_about_test.mocks.dart';
+
+
+@GenerateMocks([ArticlesRepository])
+void main() {
+  late ArticlesRepository repository;
+  late List<Article> topHeadlines;
+
+  setUp(() {
+    repository = MockArticlesRepository();
+    topHeadlines = [
+      const Article(
+          source: Source(id: "id", name: "name"),
+          author: "author",
+          title: "title",
+          description: "description",
+          url: "url",
+          urlToImage: "urlToImage",
+          publishedAt: "publishedAt",
+          content: "content")
+    ];
+  });
+
+  blocTest<ArticlesCubit, ArticlesState>(
+    'GIVEN topHeadlines is requested WHEN response is successful '
+    'THEN emits [TopHeadlinesLoading, TopHeadlinesLoaded]',
```

Comment:
Multiline description <3 

I'm going to copy this idea (I will probably even use 3 lines to increase readability)

Reply:
changed to 3 lines as well 👍